code/modules/jobs/job_types/research_director.dm

```diff
@@ -39,9 +39,17 @@
 	changed_maps = list("OmegaStation")
 
 	mail_goodies = list(
-		/obj/item/storage/box/monkeycubes = 30,
+		/obj/effect/spawner/lootdrop/aimodule_neutral = 15,
+		/obj/item/transfer_valve = 15,
+		/obj/effect/spawner/lootdrop/aimodule_harmless = 10,
+		/obj/item/clothing/mask/facehugger/toy = 5,
 		///obj/item/circuitboard/machine/sleeper/party = 3,
-		/obj/item/borg/upgrade/ai = 2
+		/obj/item/borg/upgrade/ai = 2,
+		/obj/effect/spawner/lootdrop/surgery_tool_alien = 2,
+		/obj/effect/spawner/lootdrop/engineering_tool_alien = 2,
+		/obj/effect/spawner/lootdrop/organ_spawner = 2,
+		/obj/effect/spawner/lootdrop/aimodule_harmful = 1,
+		/obj/item/stack/ore/bluespace_crystal/refined/nt = 1
```

Comment:
this stuff is the NT equivalent of TP, right?
Not sure if they can ever actualy use it, so maybe it's fine?

Reply:
sounds good